.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "org.lflang/src/lib/c/reactor-c"]
 	path = core/src/main/resources/lib/c/reactor-c
-	url = https://github.com/lf-lang/reactor-c.git
+	url = git@github.com:lf-lang/reactor-c.git
```

Comment:
This is not a good idea because anyone who checks out the repo and does not have public-key auth set up will run into trouble. This is why we use `https` for all our submodules.